falco.yaml

```diff
@@ -76,30 +77,68 @@ pipeline:
       # Replace the find_dependency with find_package macro for newer cmake, otherwise it will fail.
       sed -i 's/find_dependency(Protobuf CONFIG)/find_package(Protobuf CONFIG)/' /usr/lib64/cmake/grpc/gRPCConfig.cmake
 
-  - runs: |
-      mkdir -p "${{targets.destdir}}"/etc/falco
-      install -Dm755 ./falco.yaml "${{targets.destdir}}"/etc/falco/falco.yaml
-      sed -e 's/time_format_iso_8601: false/time_format_iso_8601: true/' < "${{targets.destdir}}"/etc/falco/falco.yaml
+data:
+  - name: builds
+    items:
+      falco-standard: Cloud Native Runtime Security; built with the driver loader
```

Comment:
I didn't see the `falco` packaged used in any images yet, and since you can't (?) have a subpackage the same name as the package I opted for `falco-standard` for the build with the driver loader.

Reply:
I'm not sure if we able to use exact same package name in its one of the sub-packages. FWIW, I would expect consumers to install with the `falco` naming rather than `falco-standard`.

Reply:
I ran into an issue when I reused the name for a subpackage, but that must have been a separate issue because I tried again and was successful with a subpackage named `falco`

Reply:
From the CI
```
unable to add edge for subpackage "falco-0.36.1-r2" from falco-0.36.1-r2: edge would create a cycle
```

Subpackage cannot be name of package. If we want the subpackage to stay `falco`, the main package will need to change.

Reply:
But if the main package name changes, how does that affect epoch and already built falco packages 🤔 

As nice as it is to reduce duplication by using subpackages, I think the easiest path forward here is to either
1. build `falco` as the main package in `falco.yaml`, and `falco-no-driver` as a subpackge, with most of the steps duplicated (replacing e.g. `targets.subpkgdir` for `targets.destdir`)
2. submit a separate `falco-no-driver.yaml` package